src/pivot-table/constants.ts

```diff
@@ -1,4 +1,5 @@
 export const DEFAULT_FONT_SIZE = "12px";
+export const DEFAULT_ICON_SIZE = "12px";
```

Comment:
Maybe rename it to `LOCK_ICON_SIZE` since it only really applicable to the lock icon?